charts/posthog/templates/_kafka.tpl

```diff
@@ -1,31 +1,48 @@
 {{/* Common Kafka ENV variables and helpers used by PostHog */}}
 
 {{/* Return the Kafka fullname */}}
-{{- define "posthog.kafka.fullname" -}}
-{{- if .Values.kafka.fullnameOverride -}}
-{{- .Values.kafka.fullnameOverride | trunc 63 | trimSuffix "-" -}}
-{{- else if .Values.kafka.nameOverride -}}
-{{- printf "%s-%s" .Release.Name .Values.kafka.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- define "posthog.kafka.fullname" }}
```

Comment:
Q: Why not remove the whitespace here?

This is a thing I've been thinking about the more I've worked with helm - we should start standardizing the whitespace approaches. There's too many to count.

Reply:
Good question! Do you have a proposal about which standard should we use?

Reply:
I think I do (always strip whitespace + use nindent). I'll create a proposal PR soonish cleaning up a few files! :)